packages/sdk/src/ZClientUtils.ts

```diff
@@ -9,34 +9,38 @@
  *
  */
 
-import * as fs from "fs";
+import { readFileSync } from "node:fs";
+import { homedir } from "node:os";
 import * as path from "node:path";
-import { homedir } from "os";
 import type { ISshSession } from "@zowe/zos-uss-for-zowe-sdk";
 import * as sshConfig from "ssh-config";
 
 export interface sshConfigExt extends ISshSession {
```

Comment:
Mostly to follow other standards throughout the codebase...
```suggestion
export interface ISshConfigExt extends ISshSession {
```

Note: this will likely trigger a chain reaction of changes.
So, I suggest selecting the word `sshConfigExt` and using VSCode's renaming utility `F2` 😋